apps/snitch_core/lib/core/domain/taxonomy/taxonomy.ex

```diff
@@ -169,6 +169,18 @@ defmodule Snitch.Domain.Taxonomy do
     Repo.all(from(taxon in Taxon, where: taxon.parent_id == ^taxon_id))
```

Comment:
@SagarKarwande refactor this to use `as_nested_set` function.

Reply:
Done